CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# How can I contribute?
+
+- You can add questions and answers in FAQ.
```

Comment:
This line was true for FAQ page. Here we do not have any. Can we please not have it here? It doesn't make sense to be here.